src/main/app/utils/featureToggles.ts

```diff
@@ -56,6 +56,10 @@ export class FeatureToggles {
     return this.launchDarklyClient.serviceVariation('dashboard_pagination_enabled', toBoolean(config.get<boolean>(`featureToggles.dashboard_pagination_enabled`)))
   }
 
+  async isEnhancedMediationJourneyEnabled (): Promise<boolean> {
```

Comment:
either we should use await in place of return or remove async from the method signature

Reply:
Ignore as the complete project follows same pattern

Reply:
thank you.